assets/js/modules/adsense/index.js

```diff
@@ -52,12 +52,8 @@ addFilter(
 	fillFilterWithComponent( DashboardZeroData )
 );
 
-let isAdBlockerActive = () => {};
-
 export const registerStore = ( registry ) => {
 	registerDatastore( registry );
-	// TODO: fix hack
-	isAdBlockerActive = () => registry.__experimentalResolveSelect( STORE_NAME ).isAdBlockerActive();
 };
```

Comment:
The `registerStore` function can be re-exported directly from `datastore` now rather than aliasing to add this hack. See other stores for how this looks like.